lib/nodes/uldaq.cpp

```diff
@@ -85,6 +85,12 @@ static const struct {
               {"unipolar-0.01", UNIPT01VOLTS, 0.0, +0.01},
               {"unipolar-0.005", UNIPT005VOLTS, 0.0, +0.005}};
 
+static const struct {
+  const char *name;
+  ScanOption clock_source;
+} clock_sources[] = {{"internal", (ScanOption)( 0 << 4)},
```

Comment:
the uldaq library doesn't have a "select internal clock" option, this is supposed to "emulate" it. (could also be shortened to just 0)